src/app/content/search/selectors.ts

```diff
@@ -27,6 +29,11 @@ export const selectedResult = createSelector(
   (state) => state.selectedResult
 );
 
+export const hits = createSelector(
+  localState,
+  (state) => state.results ? state.results.hits.hits : null
```

Comment:
Not sure, but I usually don't try to modify existing code which is not directly related to the pr because it's often leading to the rabbit hole where you spent time on trying to understand another part of the repo (which in some cases might be a good thing after all)